Updated the text for get-started-api documentation (#4055)
dapr/docs: daprdocs/content/en/getting-started/get-started-api.md

### Step 1: Run the Dapr sidecar

The [`dapr run`]({{< ref dapr-run.md >}}) command normally runs your application and a Dapr sidecar. In this case, 

> it will only run the sidecar since you will interact with the state management API directly.

Reviewer: ```suggestion
it only runs the sidecar since you are interacting with the state management API directly.
```
